### PR TITLE
Add deobfuscation uploading

### PR DESCRIPTION
**Pull request by Vacxe**
Close: #18 

Please check this PR and make a release ASAP. That's really necessary. Cheers!

### CODE REVIEW

**Comment by nprail** [comment]
Thanks for the PR! 😁 I'll review it as soon as I can (likely later today).

File: src/actions/Upload.ts [hunk 1]
```diff
@@ -86,6 +90,26 @@ export class Upload extends Edit {
...
+          media: {
+            body: createReadStream(this.uploadParams.deobfuscation),
+            mimeType: 'application/octet-stream'
+          },
+          packageName: this.editParams.packageName
+        },
+          {}
+      )
+    }
+    return Promise.resolve()
```
Comment (nprail) [inline src/actions/Upload.ts]: Returning `Promise.resolve` is a bit redundant for an async function :)
```suggestion
    }
```

File: src/actions/Upload.ts [hunk 2]
```diff
@@ -86,6 +90,26 @@ export class Upload extends Edit {
...
+          apkVersionCode: this.editParams.versionCode,
+          deobfuscationFileType: 'proguard',
+          editId: this.editId,
+          media: {
+            body: createReadStream(this.uploadParams.deobfuscation),
+            mimeType: 'application/octet-stream'
+          },
+          packageName: this.editParams.packageName
+        },
+          {}
```
Comment (nprail) [inline src/actions/Upload.ts]: Slight code-style issue 🙂 
```suggestion
        {}
```

File: src/actions/Upload.ts [hunk 3]
```diff
@@ -15,6 +15,8 @@ export interface IUploadParams {
   releaseNotes?: IReleaseNotes[]
   /** An array that specifies the paths to the expansion files (OBBs) for this release. */
   obbs?: string[]
+  /** An array that specifies the paths to the deobfuscation files for this release. */
+  deobfuscation?: string
```
Comment (nprail) [inline src/actions/Upload.ts]: Thanks for documenting it! 

Is there only ever one deobfuscation file?

Comment (Vacxe) [reply src/actions/Upload.ts]: Isn't it makes sense? One Apk - One deobfuscation

**Comment by Vacxe** [comment]
Marks fixed. Please prepare a release ASAP.

File: src/actions/Upload.ts (hunk 3 above)
Comment (nprail) [reply src/actions/Upload.ts]: Just wanted to clarify since I honestly did not know for sure :)

**Pull request merged**